missed_events_replayer/missed_events_replayer/archive.py

```diff
@@ -0,0 +1,75 @@
+import time
+
+import boto3
+
+from helpers import console_print, get_logger
+
+LOGGER = get_logger()
+DATE_RANGE_FORMAT = '%Y-%m-%d'
+MAX_WAIT_SECONDS = 600
+client = boto3.client('ecs')
+
+
+def get_matomo_container_instance_arn():
+    all_container_instance_arns = client.list_container_instances(cluster='platform-web')['containerInstanceArns']
+
+    for arn in all_container_instance_arns:
+        task_arns = client.list_tasks(cluster='platform-web', containerInstance=arn)['taskArns']
+        task_defs = client.describe_tasks(cluster='platform-web', tasks=task_arns)['tasks']
+        for task_def in task_defs:
+            if any(container['name'] == 'matomo' for container in task_def['containers']):
+                return arn
+
+    LOGGER.error(f"No container instances found running Matomo. Instance IDs: f{all_container_instance_arns}")
+    exit(1)
+
+
+def wait_and_return_succesful_command_response(ssm_client, command_id, ec2_instance_id):
+    slept_time = 0
+    while True:
+        time.sleep(1)
+        slept_time += 1
+        command_response = ssm_client.get_command_invocation(
+                CommandId=command_id, 
+                InstanceId=ec2_instance_id,
+            )
+        if command_response['Status'] == 'Success':
+            return command_response
+        if slept_time == MAX_WAIT_SECONDS:
```

Comment:
```suggestion
        if slept_time >= MAX_WAIT_SECONDS:
```
Slightly defensive I know... (and also maybe that can't happen anyway?) Just a suggestion.

Reply:
Works for me!